### PR TITLE
[master] set default compression level when creating images with dasd type

### PR DESCRIPTION
**Pull request by wanxiangyuer**
1. set default compression level 6 when creating images with dasd type.
2. modify the --help message.

### CODE REVIEW

**Comment by dongyanyang** [comment]
@wgxoyun need rebase

**Comment by wanxiangyuer** [comment]
@dongyanyang this is the test result:
capturing the volume image with default and specified compression level :
```
[root@dycomput ~]# /opt/zthin/bin/creatediskimage 5c60 0x5005076802400c1a 0x0002000000000000 test_vol_image --compression 6
Warning: --compression option is not supported when creating images from volume, will ignore it and use compression level 0.
creatediskimage start time: 2020-05-14-11:11:42.125
FCP CHANNEL:    "5c60"
WWPN:           "0x5005076802400c1a"
LUN:            "0x0002000000000000"
IMAGE FILE:     "test_vol_image"
COMPRESSION:    "0"

Creating test_vol_image image file for 0x5005076802400c1a/0x0002000000000000 disk at channel 5c60 with disk size 6291456 BLK.
Compression level: 0
Header label: 0
Image creation successful.
creatediskimage end time: 2020-05-14-11:12:13.424
[root@dycomput ~]# ls -lh test_vol_image
-rw-r--r--. 1 root root 3.0G May 14 07:12 test_vol_image
[root@dycomput ~]# /opt/zthin/bin/creatediskimage 5c60 0x5005076802400c1a 0x0002000000000000 test_vol_image
creatediskimage start time: 2020-05-14-11:39:46.476
FCP CHANNEL:    "5c60"
WWPN:           "0x5005076802400c1a"
LUN:            "0x0002000000000000"
IMAGE FILE:     "test_vol_image"
COMPRESSION:    "0"

Creating test_vol_image image file for 0x5005076802400c1a/0x0002000000000000 disk at channel 5c60 with disk size 6291456 BLK.
Compression level: 0
Header label: 0
Image creation successful.
creatediskimage end time: 2020-05-14-11:40:16.823
[root@dycomput ~]# ls -lh test_vol_image
-rw-r--r--. 1 root root 3.0G May 14 07:40 test_vol_image
```

capturing the fba image with default compression level and specified level:
```
[root@dycomput ~]# /opt/zthin/bin/creatediskimage img82fba 0100 ./rhel82_ext4.img
creatediskimage start time: 2020-05-14-11:05:53.085
SOURCE USER ID: "img82fba"
DISK CHANNEL:   "0100"
IMAGE FILE:     "./rhel82_ext4.img"
COMPRESSION:    "6"

Creating rhel82_ext4.img image file for img82fba's disk at channel 0100 with disk size 20971520 BLK.
Compression level: 6
Image creation successful.
creatediskimage end time: 2020-05-14-11:08:42.749
[root@dycomput ~]# ls -lh rhel82_ext4.img
-rw-r--r--. 1 root root 535M May 14 07:08 rhel82_ext4.img
[root@dycomput ~]# /opt/zthin/bin/creatediskimage img82fba 0100 ./rhel82_ext4.img --compression 7
creatediskimage start time: 2020-05-14-11:18:40.749
SOURCE USER ID: "img82fba"
DISK CHANNEL:   "0100"
IMAGE FILE:     "./rhel82_ext4.img"
COMPRESSION:    "7"

Creating rhel82_ext4.img image file for img82fba's disk at channel 0100 with disk size 20971520 BLK.
Compression level: 7
Image creation successful.
creatediskimage end time: 2020-05-14-11:21:35.586
[root@dycomput ~]# ls -lh rhel82_ext4.img
-rw-r--r--. 1 root root 533M May 14 07:21 rhel82_ext4.img
```

File: zthin-parts/zthin/bin/creatediskimage [hunk 1]
```diff
@@ -70,11 +70,6 @@ function parseArgs {
   if [[ -z "$gzipCompression" ]]; then
     gzipCompression=6
   fi
-
-  if [[ -z "$headerLabel" ]]; then
-    headerLabel=0
```
Comment (bjhuangr) [inline zthin-parts/zthin/bin/creatediskimage:75]: any other place reference to `headerLabel` ?

Comment (bjhuangr) [reply zthin-parts/zthin/bin/creatediskimage:75]: make sure the change not impact other place that using `headerLabel`

Comment (dongyanyang) [reply zthin-parts/zthin/bin/creatediskimage:75]: > any other place reference to `headerLabel` ?
No other place. 
This headerlabel is added by BenLei when we want to add a switch to control whether to add image header for SCSI image. That will only be referenced by volume image.

**Comment by dongyanyang** [comment]
@jichenjc pls help to review.

**Pull request merged**